### ISSUE TITLE
Nothing happens when saving CSS file or clicking auto prefix selection

### ISSUE THREAD

**Issue by Nate75** [mikaeljorhult/brackets-autoprefixer#49]
I have seen a lot of comments regarding this issue and no fix?? resetting does nothing... assuming I am doing it right just by clicking the reset button.

I am assuming that this must be fixed by now and I am just missing something??

Thx

**Comment by ivoilic** [comment]
+1

**Comment by NicoleVache** [comment]
++1

I am having the same troubles. On the newest version, reset setting ....looking for a working fix ASAP!

Thanks :)

**Comment by madhatttr** [comment]
Same here, I have been through all the closed threads on the same subject to no avail, clean reinstalled brackets and the extensions, reset settings etc...not quite sure why it isn't working. 

Currently looks like this after just resetting the settings, i do notice that there is not enabled line as other commenters have mentioned...
{
    "fonts.fontSize": "12px",
    "fonts.fontFamily": "'SourceCodePro-Medium', ＭＳ ゴシック, 'MS Gothic', monospace",
    "themes.theme": "dark-theme",
    "mikaeljorhult.brackets-autoprefixer.visualCascade": false,
    "mikaeljorhult.brackets-autoprefixer.browsers": [
        "> 1%",
        "last 2 versions",
        "Firefox ESR",
        "Opera 12.1"
    ],
    "mikaeljorhult.bracketsAutoprefixer.visualCascade": false,
    "mikaeljorhult.bracketsAutoprefixer.browsers": [
        "> 1%",
        "last 2 versions",
        "Firefox ESR",
        "Opera 12.1"
    ]
}

also i noticed in default preferences.json;

	// 
	"mikaeljorhult.bracketsAutoprefixer.browsers": [],

	// Default: false
	"mikaeljorhult.bracketsAutoprefixer.enabled": false,

	// Default: false
	"mikaeljorhult.bracketsAutoprefixer.visualCascade": false,

I am on a mac, running the 1.5 build of brackets and 0.6.1 autoprefixer...help?

**Comment by mikaeljorhult** [comment]
Please try the just released version 0.6.2.

**Comment by mikaeljorhult** [comment]
@Nate75 Do version 0.6.2/0.6.3 solve your problem?

**Comment by mikaeljorhult** [comment]
Did the new version solve your problems @NicoleVache @madhatttr @ivoilic?

**Comment by JeremyLopez** [comment]
No, mine still isn't working with version 0.6.3.  It just doesn't seem to do anything.  Do I need to include a `.json` file in the project root to tell the add-on how to behave?

**Comment by courseiam** [comment]
Hi, same for me too. Neither option seems to work for me.

**Comment by jpstack** [comment]
I just want to add that if anyone is still experiencing this issue, as I happened to randomly today, what solved it for me was going into Edit > Autoprefixer Settings, deleting all custom Browsers except ">1%" and "last 2 versions" and systematically adding them back in while attempting to autoprefix something like a display: flex rule or similar.

Eventually I found that the problem was an old iOS rule I had in there for some reason. Removed it, restored my other Browsers, and it's working again.
